src/styles/blocks/articles.css

```
@@ -12,6 +12,110 @@
     font-size: 21px;
 }
 
+@media screen and (max-width: 320px) {
+
+    .main{
```

У нас принято ставить пробел между селектором и открывающей скобкой. Попробуйте запустить `npm test`, вы увидите ошибки.

```suggestion
    .main {
```